Gemfile

```diff
@@ -155,6 +155,7 @@ group :replication, :manageiq_default do
 end
 
 group :rest_api, :manageiq_default do
+  gem "manageiq-api", :path => "../manageiq-api"
```

Comment:
Oh, I guess this will eventually be a `manageiq-plugin` huh... then in that case I guess this follows the convention.  #trollFail